pkg/apiserver/common/utils.go

```diff
@@ -123,6 +123,11 @@ func ID(userName, fsName string) string {
 	return FsPrefix + userName + "-" + fsName
 }
 
+// PVCName generate PVC name by fsID
+func PVCName(fsID string) string {
```

Comment:
Please refer to  [ConcatenatePVCName](https://github.com/PaddlePaddle/PaddleFlow/blob/develop/pkg/common/schema/fs.go#L59)